Example/Controllers/Extensions/ViewController+MiniAppUserInfoProtocol.swift

```diff
@@ -23,9 +23,11 @@ extension ViewController {
         return contactList
     }
 
-    func getAccessToken(miniAppId: String, completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void) {
+    func getAccessToken(miniAppId: String,
+                        scopes: MASDKAccessTokenScopes?,
+                        completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void) {
```

Comment:
`MATokenInfo` also needs to be updated to contain the `audience` and `scopes`. This information should be passed back to the JS side so they know what scopes they have for that token (for example, the Host App might have removed some of the scopes if they were invalid).